### ISSUE TITLE
 loss function doesn't converge.

### ISSUE THREAD

**Issue by Analysis666** [tensorlayer/hyperpose#221]
Thanks for your sharing! 
Now, my loss function doesn't converge. The loss always shake from 200 to 800.
This is my super params:
      batch_size = 16
      save_interval = 1000
      self.lr_init = 4e-5  # initial learning rate 4e-5
      self.lr_decay_every_step = 136106  # evey number of step to decay lr
      self.lr_decay_factor = 0.333  # decay lr factor
      self.weight_decay_factor = 5e-4

I have tyied to set bach_size equal to 8,32; and  set learning rate from 1e-2 to ae-7,but also shake. 

Can you take a look at this problem?

Best wishes to you!
